gradle.properties

```diff
@@ -20,7 +20,7 @@ RMEE_GRADLE_VERSION=1.1.20200614081240
 JDK_VERSION=8u202-b08
 
 #CORDA_VERSION=4.4
-CORDA_VERSION=4.5-C3C01
```

Comment:
This corda version was not found. Also it is the main reason that the build fails in github.
I changed to the one available.
Perhaps, I need another repo?